src/components/ArticlePage.jsx

```diff
@@ -78,8 +91,37 @@ export const ArticlePage = ({ users }) => {
     }
   }
 
+  function handleDeleteComment(comment_id) {
+    const commentToDelete = comments.find(
+      (comment) => comment.comment_id === comment_id
+    );
+
+    if (commentToDelete) {
+      const updatedComments = comments.filter(
+        (comment) => comment.comment_id !== comment_id
+      );
+      setComments(updatedComments);
+
+      deleteCommentById(comment_id)
+        .then(() => {
+          setDeletionMessage("Comment Successfully Deleted");
+        })
+        .catch((error) => {
+          console.log(error);
+          setComments((prevComments) => [commentToDelete, ...prevComments]);
+          setDeletionMessage("Failed to delete comment.");
+        })
+        .finally(() => {
+          setTimeout(() => {
```

Comment:
It's a little misleading that the page is reloading if you're just showing a message on screen for three seconds. While I like the message to the user that things are successful or an error, do you need that message to fill the screen and take the user away from the content at that time?